specs/spring-cloud-contract-spec-java/src/main/java/org/springframework/cloud/contract/spec/internal/MediaTypes.java

```diff
@@ -20,76 +20,248 @@
  * Contains most commonly used media types.
  *
  * @author Marcin Grzejszczak
+ * @author Tim Ysewyn
  * @since 1.0.2
  */
 public class MediaTypes {
```

Comment:
shouldn't this be `final` with a `private` constructor?

Reply:
Yes, but that would break backwards compat. We could mark it as depracted and log something to stdout?

Reply:
That's a valid point. Let's log a warning then via a logger instead of using sys out.

Reply:
Let me do the same like `RegexPatterns`, per your remark.